webapp/src/components/reaction_stream/reaction_stream.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+
+import {getEmojiImageUrl} from 'mattermost-redux/utils/emoji_utils';
+import {UserProfile} from '@mattermost/types/lib/users';
+
+import {ReactionWithUser, EmojiData} from 'src/types/types';
+import {Emojis, EmojiIndicesByUnicode} from 'src/emoji';
+import {getUserDisplayName} from 'src/utils';
+
+type Props = {
+    reactions: ReactionWithUser[],
+    currentUserID: string,
+    profiles: {[key: string]: UserProfile;},
+};
+
+const ReactionStreamList = styled.div`
+    position='absolute';
+    left=0;
```

Comment:
`:`? 
Also a suggestion: We can probably add a fixed height to this of about 75% of the viewport height as a way to artificially make the reactions "disappear" when there are many of them.

Reply:
ok tested this and doesn't look good, but that's my faulty css, we can iterate on this later

Reply:
These lines of CSS are also invalid, they should say `:` instead of `=`.

Reply:
Actually let me make a suggestion here. Try replacing these 
```
    position='absolute';
    left=0;
    bottom=100px;
```
with
```
    align-self: flex-end;
    height: 75vh;
 ```
 I think this will have the effect we want.

Reply:
oh! good catch

Reply:
yep, that seems to work better, thanks!